src/services/data_model_engine.js

```diff
@@ -11,6 +11,7 @@ import {NotFoundError, PermissionError, ValidationError} from '../errors/errors'
 import {getTimestamp} from '../utils/time_utils';
 import {pick, put} from '../utils/dict_utils';
 import allPermissions from '../utils/all_permissions';
+const pipeline = require('util').promisify(require('stream').pipeline);
```

Comment:
I don't have anything against it, but if we have imports everywhere, is it fine to use require in that one place?

Reply:
Sadly this a limitation of import. To use imports and achieve the same thing I would have to split it into three lines:
1) import utli <- would also pollute the context with something I don't need otherwise
2) import the unpromisified version of pipeline <- would also pollute context
3) the the promisification and assign it into a const <- the name of the const can conflict with the import in point 2), so one would have to be reassigned.

IMO: using `require` is just simpler and cleaner in this case